Source/WebCore/page/text-extraction/TextExtraction.cpp

```diff
@@ -316,7 +316,7 @@ static inline Variant<SkipExtraction, ItemData, URL, Editable> extractItemData(N
         return { Editable { } };
 
     if (RefPtr image = dynamicDowncast<HTMLImageElement>(element))
-        return { ImageItemData { image->src().lastPathComponent().toString(), image->altText() } };
+        return { ImageItemData { image->getURLAttribute(HTMLNames::srcAttr).lastPathComponent().toString(), image->altText() } };
```

Comment:
Are you leaving the `get` prefix in this one so that URL stays capitalized ;)?

Reply:
You know me too well. That was one reason, but another was that I was doing too much renaming in this patch and decided to cool it. I probably would have renamed it to something that didn’t start with “url” if my primary mission was renaming, but the renaming was a distraction.